src/EditorFeatures/Core.Wpf/InheritanceMargin/MarginGlyph/MemberDisplayViewModel.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Editor.Wpf;
+using Microsoft.CodeAnalysis.InheritanceMargin;
+using Microsoft.VisualStudio.Imaging.Interop;
+
+namespace Microsoft.CodeAnalysis.Editor.InheritanceMargin.MarginGlyph
+{
+    /// <summary>
+    /// View model used to display a member in MenuItem. Only used when there are multiple members on the same line.
+    /// </summary>
+    internal class MemberDisplayViewModel
+    {
+        /// <summary>
+        /// ImageMoniker for this member.
+        /// </summary>
+        public ImageMoniker ImageMoniker { get; }
+
+        /// <summary>
+        /// Display content in the MenuItem.
+        /// </summary>
+        public string DisplayName { get; }
+
+        /// <summary>
+        /// ToolTip for the MenuItem.
+        /// </summary>
+        public string ToolTip { get; }
```

Comment:
what are the tooltips we have here?  is it hte doc comments for a symbol?

Reply:
I'll leave this open, later I could give a picture for this

Reply:
![image](https://user-images.githubusercontent.com/24360909/112598863-673fa080-8dcc-11eb-9a6d-fe52c585aa1c.png)

Do you feel the tooltip is useless since it is already selected? It could just be the automation text

Reply:
yes.  that tooltip is not necessary.  the arrow and general context-menu nature of this makes this clear, so the tooltip would just be clutter imo.

Reply:
I agree with Cyrus but also this is a really rare case.

Reply:
OK, I will remove the tooltip